apps/galleries_institutions/components/filter_facet/partner_filter_facet.coffee

```diff
@@ -35,9 +36,10 @@ module.exports = class PartnerFilterFacet extends Backbone.Model
     escape= (s) ->
       s.replace /[-\/\\^$*+?.()|[\]{}]/g, '\\$&'
 
-    regex = _s.clean(escape(query)).replace(' ', '\\W* \\W*')
+    foldedQuery = AsciiFolder.fold(query)
+    regex = _s.clean(escape(foldedQuery)).replace(' ', '\\W* \\W*')
     substrRegex = new RegExp(regex, 'i')
-    substrRegex.test string
+    substrRegex.test AsciiFolder.fold(string)
 
```

Comment:
I don't think you need to call fold twice maybe. I think you could delete `foldedQuery = AsciiFolder.fold(query)` all together.

```js
regex = _s.clean(escape(query)).replace(' ', '\\W* \\W*')
substrRegex = new RegExp(regex, 'i')
substrRegex.test AsciiFolder.fold(string)
```